### ISSUE TITLE
Running MentalOmega on WINE in 64 bit mac

### ISSUE THREAD

**Issue by sneakers-the-rat** [CnCNet/xna-cncnet-client#541]
Hello and thank you all for this wonderful work, i am really excited to try it out after seeing the trailers for mentalomega. 

I have looked through these issues:
- https://github.com/CnCNet/xna-cncnet-client/issues/326
- https://github.com/CnCNet/xna-cncnet-client/issues/29
- https://github.com/CnCNet/xna-cncnet-client/issues/59

and am reaching a different error state - I can get the main menu to open up with `wine Resources/clientxna.exe` , but trying to run any of the campaign missions spits me back out with an error dialogue 

```
Could not run executable

Message 0xgamemd.exe not found in file %2.
```

`gamemd.exe` is present (have followed the install instructions), so am not quite sure how to proceed!

Attachking the console output, though i dont' know how enlightening it will be:

<details>
<summary>expand/collapse console log</summary>

```
007c:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
0074:err:sync:RtlpWaitForCriticalSection section 00006FFFFFFAC4A0 "/Users/gcenx/Documents/GitHub/codeweavers-wine/dlls/ntdll/loader.c: loader_section" wait timed out in thread 0074, blocked by 007c, retrying (60 sec)
007c:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
007c:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
007c:fixme:hid:handle_IRP_MN_QUERY_ID Unhandled type 00000005
0024:err:environ:init_peb starting L"C:\\MentalOmega\\Resources\\clientxna.exe" in experimental wow64 mode
0024:fixme:ntdll:NtQuerySystemInformation info_class SYSTEM_PERFORMANCE_INFORMATION
0024:fixme:wow:wow64_NtQueryInformationToken unsupported class 37
0024:fixme:nls:GetFileMUIPath stub: 0x10, L"C:\\windows\\system32\\tzres.dll", (null), 0012EED8, 0001E540, 0012EEDC, 0012EED0
0024:fixme:nls:GetFileMUIPath stub: 0x10, L"C:\\windows\\system32\\tzres.dll", (null), 0012EED8, 0001E540, 0012EEDC, 0012EED0
0024:fixme:nls:GetFileMUIPath stub: 0x10, L"C:\\windows\\system32\\tzres.dll", (null), 0012EED8, 0001E540, 0012EEDC, 0012EED0
012c:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
012c:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
0130:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
0024:fixme:system:NtUserQueryDisplayConfig flags 0x2, paths_count 0x12ed0c, paths 0x336df68, modes_count 0x12ed18, modes 0x2210cf0, topology_id 0x0 semi-stub
012c:fixme:wbemprox:wbem_locator_ConnectServer authentication not supported
0130:fixme:wbemprox:wbem_locator_ConnectServer authentication not supported
012c:fixme:wbemprox:wbem_locator_ConnectServer specific locale not supported
0130:fixme:wbemprox:wbem_locator_ConnectServer specific locale not supported
012c:fixme:wbemprox:wbem_locator_ConnectServer unsupported flags
0130:fixme:wbemprox:wbem_locator_ConnectServer unsupported flags
012c:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
0130:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
0130:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
0130:fixme:wbemprox:wbem_locator_ConnectServer authentication not supported
0130:fixme:wbemprox:wbem_locator_ConnectServer specific locale not supported
0130:fixme:wbemprox:wbem_locator_ConnectServer unsupported flags
012c:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
012c:fixme:wbemprox:wbem_locator_ConnectServer authentication not supported
012c:fixme:wbemprox:wbem_locator_ConnectServer specific locale not supported
012c:fixme:wbemprox:wbem_locator_ConnectServer unsupported flags
0130:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
0024:fixme:win:RegisterTouchWindow hwnd 00020074, flags 0x3 stub!
0024:fixme:wgl:macdrv_wglChoosePixelFormatARB unused pfAttribFList
012c:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
012c:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
012c:fixme:wbemprox:wbem_locator_ConnectServer authentication not supported
012c:fixme:wbemprox:wbem_locator_ConnectServer specific locale not supported
012c:fixme:wbemprox:wbem_locator_ConnectServer unsupported flags
012c:fixme:ole:thread_context_info_QueryInterface interface not implemented {51372ae0-cae7-11cf-be81-00aa00a2fa25}
/vagrant/mono/mono/eglib/gpath.c:115: assertion 'filename != NULL' failed

/vagrant/mono/mono/eglib/gpath.c:115: assertion 'filename != NULL' failed

0024:fixme:msctf:ThreadMgr_ActivateEx Unimplemented flags 0x4
0024:fixme:imm:ImeSetCompositionString himc 00030050, index 9, comp 79B71D48, comp_len 2, read 79B71D48, read_len 2 semi-stub!
0024:fixme:imm:ImeSetCompositionString Read string unimplemented
0024:fixme:imm:NotifyIME himc 00030050, action 0x11, index 0, value 0 stub!
0024:fixme:coreaudio:ca_channel_layout_to_channel_mask Unhandled channel 0xffffffff
0024:fixme:coreaudio:ca_channel_layout_to_channel_mask Unhandled channel 0xffffffff
0024:fixme:coreaudio:ca_channel_layout_to_channel_mask Unhandled channel 0xffffffff
0024:fixme:coreaudio:ca_channel_layout_to_channel_mask Unhandled channel 0xffffffff
0024:fixme:coreaudio:ca_channel_layout_to_channel_mask Unhandled channel 0xffffffff
0024:fixme:coreaudio:ca_channel_layout_to_channel_mask Unhandled channel 0xffffffff
0024:fixme:coreaudio:ca_channel_layout_to_channel_mask Unhandled channel 0xffffffff
0058:fixme:nsi:ipv6_forward_enumerate_all not implemented
0058:fixme:nsi:ipv6_forward_enumerate_all not implemented
0058:fixme:nsi:ipv6_forward_enumerate_all not implemented
0058:fixme:nsi:ipv6_forward_enumerate_all not implemented
0024:fixme:coreaudio:ca_channel_layout_to_channel_mask Unhandled channel 0xffffffff
0024:fixme:coreaudio:ca_channel_layout_to_channel_mask Unhandled channel 0xffffffff

(wine:60243): GStreamer-WARNING **: 23:28:51.835: Failed to load plugin '/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0/libgstopenjpeg.dylib': dlopen(/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0/libgstopenjpeg.dylib, 0x0002): Library not loaded: @rpath/@rpath/lib/libopenjp2.7.dylib
  Referenced from: <27E10BE3-513F-3BC4-83EE-D61465333772> /Library/Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0/libgstopenjpeg.dylib
  Reason: tried: './@rpath/lib/libopenjp2.7.dylib' (no such file), '/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0/../../@rpath/lib/libopenjp2.7.dylib' (no such file), '/Applications/Wine Stable.app/Contents/Resources/@rpath/lib/libopenjp2.7.dylib' (no such file), '/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0/../lib/@rpath/lib/libopenjp2.7.dylib' (no such file), '/Applications/Wine Stable.app/Contents/Resources/wine/lib/@rpath/lib/libopenjp2.7.dylib' (no such file), '/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0/../@rpath/lib/libopenjp2.7.dylib' (no such file), '/Applications/Wine Stable.app/Contents/Resources/wine/@rpath/lib/libopenjp2.7.dylib' (no such file), './@rpath/lib/libopenjp2.7.dylib' (no such file), '/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0/../../@rpath/lib/libopenjp2.7.dylib' (no such file), '/Applications/Wine Stable.app/Contents/Resources/@rpath/lib/libopenjp2.7.dylib' (no such file), '/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0/../lib/@rpath/lib/libopenjp2.7.dylib' (no such file), '/Applications/Wine Stable.app/Contents/Resources/wine/lib/@rpath/lib/libopenjp2.7.dylib' (no such file), '/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/gstreamer-1.0/../@rpath/lib/libopenjp2.7.dylib' (no such file), '/Applications/Wine Stable.app/Contents/Resources/wine/@rpath/lib/libopenjp2.7.dylib' (no such file), './@rpath/lib/libopenjp2.7.dylib' (no such file), '/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/../@rpath/lib/libopenjp2.7.dylib' (no such file), '/Applications/Wine Stable.app/Contents/Resources/wine/@rpath/lib/libopenjp2.7.dylib' (no such file), '/Library/Frameworks/GStreamer.framework/Versions/1.0/lib/../lib/@rpath/lib/libopenjp2.7.dylib' (no such file), '/Applications/Wine Stable.app/Contents/Resources/wine/lib/@rpath/lib/libopenjp2.7.dylib' (no such file)
0024:fixme:imm:ImeSetCompositionString himc 00030050, index 9, comp 79B71D48, comp_len 2, read 79B71D48, read_len 2 semi-stub!
0024:fixme:imm:ImeSetCompositionString Read string unimplemented
0024:fixme:imm:NotifyIME himc 00030050, action 0x11, index 0, value 0 stub!
0024:fixme:exec:SHELL_execute flags ignored: 0x00004100
01a8:err:environ:init_peb starting L"C:\\windows\\syswow64\\winebrowser.exe" in experimental wow64 mode
0024:fixme:imm:ImeSetCompositionString himc 00030050, index 9, comp 79B71D48, comp_len 2, read 79B71D48, read_len 2 semi-stub!
0024:fixme:imm:ImeSetCompositionString Read string unimplemented
0024:fixme:imm:NotifyIME himc 00030050, action 0x11, index 0, value 0 stub!
01c4:err:environ:init_peb starting L"C:\\MentalOmega\\Syringe.exe" in experimental wow64 mode
0024:fixme:imm:ImeSetCompositionString himc 00030050, index 9, comp 79B71D48, comp_len 2, read 79B71D48, read_len 2 semi-stub!
0024:fixme:imm:ImeSetCompositionString Read string unimplemented
0024:fixme:imm:NotifyIME himc 00030050, action 0x11, index 0, value 0 stub!
0024:fixme:mfplat:bytestream_file_Close 095DDBA4
01cc:err:environ:init_peb starting L"C:\\MentalOmega\\gamemd.exe" in experimental wow64 mode

01c4:fixme:ver:GetCurrentPackageId (0012FEA4 00000000): stub
0024:fixme:imm:ImeSetCompositionString himc 00030050, index 9, comp 79B71D48, comp_len 2, read 79B71D48, read_len 2 semi-stub!
0024:fixme:imm:ImeSetCompositionString Read string unimplemented
0024:fixme:imm:NotifyIME himc 00030050, action 0x11, index 0, value 0 stub!
ls
0024:fixme:mfplat:bytestream_file_Close 095DDBA4
026c:err:environ:init_peb starting L"C:\\MentalOmega\\Syringe.exe" in experimental wow64 mode
0024:fixme:imm:ImeSetCompositionString himc 00030050, index 9, comp 79B71D48, comp_len 2, read 79B71D48, read_len 2 semi-stub!
0024:fixme:imm:ImeSetCompositionString Read string unimplemented
0024:fixme:imm:NotifyIME himc 00030050, action 0x11, index 0, value 0 stub!
0024:fixme:mfplat:bytestream_file_Close 09595D5C
0278:err:environ:init_peb starting L"C:\\MentalOmega\\gamemd.exe" in experimental wow64 mode
```

</details>

Opening a new issue because i figure the situation has changed a bit since mac is now 64 bit only, and the wow64 mode is the only available mode for wine. I'm a newbie trying to emulate windows, so any input or advice would be welcome :)

**Comment by MahBoiDeveloper** [comment]
I guess the best option for your will be dual boot with Windows 10.
